test/e2e/gardener/upgrade/upgrade.go

```diff
@@ -119,4 +164,167 @@ var _ = Describe("Gardener upgrade Tests for", func() {
 			})
 		})
 	})
+
+	Context("Shoot::e2e-upgrade-ha", Label("high-availability"), func() {
```

Comment:
Can you also add a comment/log somewhere that this shoot is used for two types of upgrades simultaneously: gardener version upgrade as well as upgrading from non-HA to HA shoot?